lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -191,7 +191,7 @@ defmodule ExUnit.Case do
       config :logger, backends: []
   """
 
-  @reserved [:case, :test, :file, :line, :registered]
+  @reserved ~w(case file line test async registered)a
```

Comment:
Maybe change `@reserved` definition the other way around: use `[...]` in callbacks.ex file?